test/base.spec.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/*
+const { run, assert } = require('./util');
+
+describe('commands', () => {
+  it('should fail when invalid command is specified', () => {
```

Comment:
Lets [skip this test](https://mochajs.org/#inclusive-tests), rather then comment it out.

Reply:
Oeh, didn't know that was possible with mocha. Will make the changes

Reply:
Done